charts/seqr/templates/check-for-new-samples-from-pipeline.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "seqr.fullname" . }}-check-new-samples-job
+  labels:
+    {{- include "seqr.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": post-install, post-upgrade
+    "helm.sh/hook-weight": "-1"
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+    {{- with .Values.podAnnotations }}
+      {{- toYaml . | nindent 4 }}
+    {{- end }}
+spec:
+  template:
+    metadata:
+      labels:
+        {{- include "seqr.labels" . | nindent 8 }}
+    spec:
+      {{- with .Values.volumes }}
+      volumes:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      restartPolicy: Never
+      containers:
+        - name: {{ include "seqr.fullname" . }}-check-new-samples-job-pod
+          image: {{ $.Values.image.repository }}:{{ $.Values.image.tag | default $.Chart.AppVersion }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{- with .Values.volumeMounts }}
+          volumeMounts:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          command: ["python", "manage.py", "check_for_new_samples_from_pipeline"]
```

Comment:
will this pod have access to the `datasetVersions` values from the hail-search helm charts?

Reply:
great question... I think the answer is actually no.  The `datasetVersions` values are isolated to the `hail-search` [chart](https://github.com/broadinstitute/tgg-helm/pull/27/files#diff-a599882979d776a558dea4bcfe42688d2b053e2d88fe3598a8888aa3e398ca75R160-R177) right now.

It does look like [global values](https://helm.sh/docs/chart_template_guide/subcharts_and_globals/#global-chart-values) are an option!

Will the `seqr` pod need to compare versions?

Reply:
yeah, otherwise I can;t think of how to know which metadata to check

Reply:
👍 I'll try the `globals`.

Reply:
Ok, seems to work ok (haven't built both charts together yet from `tag-helm` but I'm optimistic)!  I added a `for` loop as well, so one job per `refgenome/datasetType/rundir` tuple.